NodeBase/entry_point.sh

```diff
@@ -32,10 +32,13 @@ fi
 # TODO: Look into http://www.seleniumhq.org/docs/05_selenium_rc.jsp#browser-side-logs
 
 SERVERNUM=$(get_server_num)
+
+rm -f /tmp/.X*lock
+
 xvfb-run -n $SERVERNUM --server-args="-screen 0 $GEOMETRY -ac +extension RANDR" \
   java ${JAVA_OPTS} -jar /opt/selenium/selenium-server-standalone.jar \
     -role node \
-    -hub http://$HUB_PORT_4444_TCP_ADDR:$HUB_PORT_4444_TCP_PORT/grid/register \
+    -hub http://hub:4444/grid/register \    
```

Comment:
I made the changes as per https://github.com/SeleniumHQ/docker-selenium/pull/238